modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,228 @@
+= Topics as Iceberg Tables
+:description: Learn how to integrate Redpanda topics with Apache Iceberg.
+:page-context-links: [{"name": "Linux", "to": "manage:topic-iceberg-integration.adoc" } ]
+:page-categories: Management, High Availability, Data Replication, Integration
+:page-beta: true
+
+
+The Apache Iceberg integration for Redpanda allows you to store topic data in the cloud in Iceberg's table format. This makes your streaming data immediately available for analytical systems, such as data warehouses like RedShift, Snowflake, and Clickhouse, and big data processing platforms, such as Apache Spark and Flink, without setting up and maintaining additional ETL pipelines. 
+
+The Iceberg integration uses xref:manage:tiered-storage.adoc[Tiered Storage]. When a cluster or topic has Tiered Storage enabled, Redpanda stores the Iceberg files in the configured Tiered Storage bucket or container.  
+
+== Prerequisites
+
+* Install `rpk`.
+
+*  {empty}
+include::shared:partial$enterprise-license.adoc[]
++
+To check if you already have a license key applied to your cluster:
++
+[,bash]
+----
+rpk cluster license info
+----
+
+* Enable xref:manage:tiered-storage.adoc#set-up-tiered-storage[Tiered Storage] for the topics for which you want to generate Iceberg tables.
+
+== Limitations
+
+* You can only enable Iceberg topic integration for new Redpanda topics. 
+* It is not possible to append or backfill data from Redpanda topics to an existing Iceberg table.
+* JSON schemas are not currently supported. For Avro schemas, records cannot contain fields greater than 128 KB.
+* You can only use one schema per topic. Schema versioning as well as upcasting (where a value is cast into its more generic data type) are not supported. 
+
+== Architecture 
+
+Apache Iceberg is an open source format specification for defining structured tables in a data lake. The table format lets you quickly and easily manage, query, and process huge amounts of structured and unstructured data. This is similar to the way in which you would manage and run SQL queries against relational data in a database or data warehouse. The open format lets you use many different languages, tools, and applications to process the same data in a consistent way, so you can avoid vendor lock-in. This data management system is referred to as a _data lakehouse_.
+
+In the Iceberg specification, tables comprise the following:
```

Comment:
I think we mean," tables are composed of" , or ' multiple layers of files comprise a table'